Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py

```diff
@@ -48,7 +48,7 @@ def install_custom_pack(pack_id: str, skip_verify: bool, skip_validation: bool,
         context_files = [context_files]
 
     for file_in_context in context_files:
-        if file_in_context['Name'] == f'{pack_id}.zip':
+        if file_in_context['Name'].split('/')[-1] == f'{pack_id}.zip':
```

Comment:
```suggestion
        if file_in_context.get('Name', '').split('/')[-1] == f'{pack_id}.zip' or file_in_context.get('Name', '') == f'{pack_id}.zip':
```

Let's ensure that we don't create any BC to any other user.